Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/README.md

```diff
@@ -4740,6 +4741,309 @@ Retrieves all container audit events when a runtime sensor such as process, netw
 >| b5 | n7 | na6 | img6 | block | filesystem | malwareFileFeed | high |
 
 
+### prisma-cloud-archive-audit-incident
+
+***
+Acknowledges an incident and moves it to an archived state.
+
+#### Base Command
+
+`prisma-cloud-archive-audit-incident`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| incident_id | Incident ID. | Required | 
+| action | Action for the command. archive - incident will be archived, unarchive - incident will be unarchived. Possible values are: archive, unarchive. Default is archive. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| PrismaCloudCompute.RuntimeContainerAuditEvents.test | String | The operating system of the container. | 
+
+#### Command example
+```!prisma-cloud-archive-audit-incident incident_id="1111"```
+#### Human Readable Output
+
+>Incident 1111 was successfully archived
+
```

Comment:
I think there should also be a context example here no?

Reply:
It doesn't save anything, just prints the result

Reply:
so your integration ymk should be fixed